### ISSUE TITLE
Table sort arrow position

### ISSUE THREAD

**Issue by Chrriis** [JFormDesigner/FlatLaf#34]
Hi,

With system L&F on Windows 10, the sort icon for tables is placed at the top of the header text.
This is actually very useful, because it does not change the minimum necessary width of the column to display the header text and thus allows more compact columns.

Here is a screenshot showing the behavior on Windows 10 L&F and with FlatLaf for comparison: 
![SortArrowPosition](https://user-images.githubusercontent.com/1174202/72048882-13779300-32be-11ea-8364-0ef4f17a63e0.png)

Do you think it would be possible to implement a position attribute to specify left/right/top/bottom ? Note that I don't know if that would fit with right-to-left logic but you get the idea 🙂

**Comment by DevCharly** [comment]
Yes, that should be not that difficult to implement and is a nice improvement 👍

**Comment by DevCharly** [comment]
This is now implemented in master branch.

![image](https://user-images.githubusercontent.com/5604048/73693442-86fe9b80-46d6-11ea-9c3b-90c1770e8616.png)

To enable it use:

~~~java
UIManager.put( "TableHeader.sortIconPosition", "top" );
~~~

"bottom" and "left" are also supported:
![image](https://user-images.githubusercontent.com/5604048/73693527-b2818600-46d6-11ea-8dfc-7920b004db0d.png) ![image](https://user-images.githubusercontent.com/5604048/73693537-b6ada380-46d6-11ea-9a86-efd9d3b135c0.png)

**Comment by Chrriis** [comment]
@DevCharly Thanks for the fix, it is almost perfect! 🙂 When the arrow is at the top, I feel it is too close to the text. I think it would look better if it were 1 pixel up.
See this comparison "now vs 1px up":
![FlatLafTopSortArrowPosition](https://user-images.githubusercontent.com/1174202/73723421-4a0fc480-4729-11ea-88d2-79361e1e240f.png)

**Comment by Chrriis** [comment]
I am not concerned by right-to-left support. Still, I am curious: how does the table behave in RTL mode, and what does "left"/"right" sort position mean in that case? Should you also support "leading" and "trailing" positions then?

**Comment by DevCharly** [comment]
Thanks for your feedback. I've optimized the position of the top arrow.
Also fixed the position of the chevron style arrow, which was not at the same position as the triangle arrow.

Regarding RTL: this is indeed not supported. I've tested all system LaFs (on Windows and macOS) and some popular open-source LaFs, but all of them ignore RTL for the sort arrow position. So right now, I think I leave it as it is...
